src/adminDashboard/adminDashboard.ts

```diff
@@ -1,17 +1,23 @@
-import { EthereumService } from "services/EthereumService";
-import { Seed } from "entities/Seed";
+import BigNumber from '../services/BigNumberService';
```

Comment:
Why is this here?  In any case, unless under certain circumstances, you should use the ethers.js version of BigNumber, not this service.

Reply:
done, removed the unnecessary imports

Reply:
Will make sure to use ethers.js